### PR TITLE
Add myself

### PR DESCRIPTION
**Pull request by tobiasahlin**
Wow that’s cool

### CODE REVIEW

**Comment by tobiasahlin** [comment]
@abergs :eyes:

**Comment by abergs** [comment]
Naah, I don't like this...

**Comment by abergs** [comment]
@tobiasahlin Could you add your picture?

File: _people/tobi.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+fullname: Tobias Ahlin
```
Comment (tobiasahlin) [inline _people/tobi.md]: Hey that's a weird name?

File: _people/tobi.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+twitter: tobiasahlin
+image: bengt.gif
```
Comment (abergs) [inline _people/tobi.md]: This is not you.

**Comment by evalinns** [comment]
Hoho

**Comment by DenaEidi** [comment]
+---

I have stuff to say also.

**Pull request merged**